src/foam/util/UIDGenerator.js

```diff
@@ -55,103 +54,50 @@ foam.CLASS({
       synchronized: true,
       type: 'String',
       documentation: `
-        Generate a Unique ID. The Unique ID consists of : 8 hexits timestamp(s) + at least 2 hexits sequence inside second 
-        + 2 hexits checksum. After the checksum is added, the ID is permutated based on the permutationSeq. In most cases, 
-        the generated ID should be 12 digits long.
+        Generate a Unique ID. The Unique ID consists of :
+        8 hexits timestamp(s) + at least 2 hexits sequence inside second + 3 hexits checksum.
+
+        After the checksum is added, the ID is permutated based on the
+        permutationSeq. In most cases, the generated ID should be 13 digits long.
       `,
       javaCode: `
+        var id = new StringBuilder();
+        var support = UIDSupport.instance();
```

Comment:
Is this worth it? Why not just use UIDSupport or import UIDSupport.toHexString()?